### DISCUSSION TITLE
Jira ticket as branch name

### DISCUSSION THREAD

**Discussion by ankitdadhich** [renovatebot/renovate#15935] [Request Help] [answered]
We have branch name restrictions to be same as Jira ticket id. Current configuration it picks naming from branch topic **renovate/configure**. This does not work in our case.
Is it possible to create Jira ticket as branch name?
I.e Jira ticket is **ABC-123**, newly created branch shall be **ABC-123**

**Comment by HonkingGoose** [comment]
I don't get _why_ you want to change the default branch names that Renovate uses? Renovate bot doesn't close any issues on your project. 😉

**Comment by rarkins** [comment, marked as answer]
Renovate needs to create many branches, so it's not really feasible to use just one branch name
